airbyte-workers/src/main/java/io/airbyte/workers/config/ProcessFactoryBeanFactory.java

```diff
@@ -118,10 +122,12 @@ public ProcessFactory discoverDockerProcessFactory(
   public ProcessFactory discoverKubernetesProcessFactory(
                                                          @Named("discoverWorkerConfigs") final WorkerConfigs workerConfigs,
                                                          @Value("${airbyte.worker.job.kube.namespace}") final String kubernetesNamespace,
+                                                         @Value("${airbyte.worker.job.kube.service.account") final String kubernetesServiceAccount,
```

Comment:
@benmoriceau left a comment:

> This value is missing from the file application.yaml in the resource folder of the airbyte-worker module. This will cause some issue while being created.